### ISSUE TITLE
Possible VerifyCsrfToken Solution

### ISSUE THREAD

**Issue by mikebronner** [Blogify/Blogify#11]
I have a project with a similar need, where I needed to "whitelist" Ajax calls from Csrf. The solution I used was to update the Middleware with the required ruleset, instead of replacing the Csrf Middleware completely (as this may cause other issues if an existing project already uses customized Csrf middleware.

So my original VerifyCsrfToken middleware:

``` php
class VerifyCsrfToken extends BaseVerifier {

    /**
     * Handle an incoming request.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \Closure  $next
     * @return mixed
     */
    public function handle($request, Closure $next)
    {
        return parent::handle($request, $next);
    }
}
```

Became:

``` php
class VerifyCsrfToken extends BaseVerifier {

    /**
     * Handle an incoming request.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \Closure  $next
     * @return mixed
     */
    public function handle($request, Closure $next)
    {
            if ($request->is('mixpanel/webhooks/*')) {
                return $next($request);
            }

        return parent::handle($request, $next);
    }
}
```

Where `mixpanel/webhooks/*` is the "namespace" in the routes that handles all my Ajax calls for this particular item.
This way only the web hook endpoints are left "exposed", not compromising the entire app. :)

**Comment by mikebronner** [comment]
Oh, here is a timely article: https://laravel-news.com/2015/06/excluding-routes-from-the-csrf-middleware/?utm_content=buffer35fcb&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer

**Comment by jorenvh** [comment]
I saw the article to, but Blogify is currently not there yet for 5.1, at least it will work in 5.1 but it's not optimized for it. I take a look at your solution or maybe I just upgrade the package to 5.1.

**Comment by jorenvh** [comment]
This feature is now available in Laravel 5.2.
